containers/podman-desktop
# feat(extension-api): Authentication implementation

### What does this PR do?

### Screenshot/screencast of this PR



Fix #1702 

### How to test this PR?

Build and install Red Hat SSO Authentication extension from 
https://github.com/redhat-developer/podman-desktop-redhat-account-ext

Use tray menu 'Red Hat/Sing In' to trigger login to Red Hat SSO. Sign In menu getting disabled and Sing Out activated.
There is no UI to see Authentication Providers installed, which is part of other PR.
